src/renderer/base/renderer.cpp

```diff
@@ -704,7 +706,16 @@ void Renderer::_PaintBufferOutputHelper(_In_ IRenderEngine* const pEngine,
             // We'll be changing the persistent one as we run through the inner loops to detect
             // when a run changes, but we will still need to know this color at the bottom
             // when we go to draw gridlines for the length of the run.
-            const auto currentRunColor = color;
+            auto currentRunColor = color;

```

Comment:
I'd want to make sure @miniksa gets a chance to confirm the the changes here in the renderer won't have too much of a negative perf impact, at least not in the "no hyperlinks" case. From my read of it, I think this should be fine, but he's the expert here